indico/modules/categories/client/js/components/CategoryModeration.jsx

```diff
@@ -16,16 +17,26 @@ import {handleSubmitError} from 'indico/react/forms';
 import {useIndicoAxios} from 'indico/react/hooks';
 import {Translate} from 'indico/react/i18n';
 import {indicoAxios} from 'indico/utils/axios';
-import {serializeDate} from 'indico/utils/date';
 import './CategoryModeration.module.scss';
 
+function getEventTime({startDt, endDt}) {
```

Comment:
I think we need to finally figure out why the moment locale isn't always taken into account... because right now this one always uses the `en` locale, so it's using 12h format even though almost none of our current locales in Indico use that format...